### PR TITLE
netuser_buffer_test: Change the path of exp for rhel.10

### PR DESCRIPTION
**Pull request by yanglei-rh**
Change the path of exp for rhel.10,because it uses the same script as rhel9, change it to the default path and change the name to rhel/exp.c.

ID:2477
Signed-off-by: Lei Yang leiyang@redhat.com

### CODE REVIEW

**Comment by yanglei-rh** [comment]
Test Pass
(1/1) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.netuser_buffer_test.q35: PASS (647.35 s)
(1/1) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.5.0.x86_64.io-github-autotest-qemu.netuser_buffer_test.q35: PASS (723.51 s)

**Comment by yanglei-rh** [comment]
Hi @PaulYuuu Could you please help me review this patch? Thanks in advance.

File: qemu/tests/cfg/netuser_buffer_test.cfg [hunk 1]
```diff
@@ -13,6 +13,6 @@
     mtu_value = 12000
     set_mtu_cmd = "ip link set %s mtu ${mtu_value}"
     check_mtu_cmd = "ip link show dev %s"
-    exp_path = netuser/rhel8/exp.c
-    RHEL.9:
-        exp_path = netuser/rhel9/exp.c
+    exp_path = netuser/rhel9/exp.c
```
Comment (PaulYuuu) [inline qemu/tests/cfg/netuser_buffer_test.cfg:16]: I think can change the name to rhel/exp.c and use it for all disto, only use other name for specific, like RHEL8.

Comment (yanglei-rh) [reply qemu/tests/cfg/netuser_buffer_test.cfg]: Good suggestions, I have changed it, please help review it  again, thanks a lot.

**Review by PaulYuuu — APPROVED** [review]

**Comment by yanglei-rh** [comment]
> I checked the netuser dir tree, and with this PR, it seems we can remove rhel9/exp.c, also consider let the latest exp.c in netuser/exp.c?
> 
> ```
> netuser/
> ├── exp.c
> ├── rhel8
> │   └── exp.c
> └── rhel9
>     └── exp.c
> ```

Let me remove the `rhel9/exp.c`

**Comment by yanglei-rh** [comment]
Hi @PaulYuuu Please help me review it again, I have removed `rhel9/exp.c` . Thanks a lot.

**Review by PaulYuuu — APPROVED** [review]

**Comment by yanglei-rh** [comment]
Test pass:
 (1/2) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.netuser_buffer_test.q35: PASS (657.51 s)
 (2/2) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.5.0.x86_64.io-github-autotest-qemu.netuser_buffer_test.q35: PASS (612.47 s)

**Pull request merged**